tests/linearizable/ctest/nemesis.c

```diff
@@ -0,0 +1,277 @@
+#include <stdlib.h>
```

Comment:
Nemesis functionality from a C program.  I use this utility to study the effect of network partitions (and to measure recovery time) against a running cluster.